python/cudf/dataframe/column.py

```diff
@@ -281,7 +281,7 @@ def to_gpu_array(self, fillna=None):
 
         Parameters
         ----------
-        fillna : str or None
+        fillna : str or None or scalar
```

Comment:
Should be changed to `scalar, 'pandas', or None` - `str` is not actually accepted.

Reply:
sure doing it right now